packages/editor/src/components/post-sticky/index.js

```diff
@@ -21,12 +22,18 @@ export default function PostSticky() {
 
 	return (
 		<PostStickyCheck>
-			<CheckboxControl
-				__nextHasNoMarginBottom
-				label={ __( 'Stick to the top of the blog' ) }
-				checked={ postSticky }
-				onChange={ () => editPost( { sticky: ! postSticky } ) }
-			/>
+			<PostPanelRow label={ __( 'Sticky' ) }>
+				<ToggleControl
+					className="editor-post-sticky__toggle-control"
+					label={
+						<VisuallyHidden>
+							{ __( 'Stick to the top of the blog' ) }
+						</VisuallyHidden>
```

Comment:
I thought most of the components supported the `hideLabelFromVision` prop.

Reply:
Mmm good point, I could update the component to support this prop. 🤔

Reply:
It's not a blocker for this PR. Let's create a separate issue for it.